pkg/syncer/util.go

```diff
@@ -364,3 +385,42 @@ func SyncerInitConfigInfo(ctx context.Context) (*cnsconfig.ConfigurationInfo, er
 	}
 	return configInfo, nil
 }
+
+// getVcHostAndVolumeManager returns VC host and the corresponding volume manager that can access the given volume on VC.
+// In case of a single VC setup, we simply return the metadataSyncer's host and volumeManager fields.
+func getVcHostAndVolumeManager(ctx context.Context, metadataSyncer *metadataSyncInformer, volumeID string) (string, volumes.Manager, error) {
```

Comment:
Can you rename this from `getVcHostAndVolumeManager` to `getVcHostAndVolumeManagerForVolumeID`